GPU/Common/FragmentShaderGenerator.cpp

```diff
@@ -81,12 +81,7 @@ bool GenerateFragmentShader(const FShaderID &id, char *buffer, const ShaderLangu
 	bool doFlatShading = id.Bit(FS_BIT_FLATSHADE) && !bugs.Has(Draw::Bugs::BROKEN_FLAT_IN_SHADER);
 	bool shaderDepal = id.Bit(FS_BIT_SHADER_DEPAL);
 	bool bgraTexture = id.Bit(FS_BIT_BGRA_TEXTURE);
-	bool colorWriteMask = id.Bit(FS_BIT_COLOR_WRITEMASK);
-
-	if (colorWriteMask && !compat.bitwiseOps) {
-		*errorString = "Color Write Mask requires bitwise ops";
-		return false;
-	}
+	bool colorWriteMask = id.Bit(FS_BIT_COLOR_WRITEMASK) && compat.bitwiseOps;
```

Comment:
I intended this to be checked during ID generation so the failure case should not happen, but apparently didn't implement that huh, good catch. Not sure where it makes the most sense to do the check, but this is fine.